flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-doris/src/test/java/org/apache/flink/cdc/connectors/doris/sink/utils/DorisSinkTestBase.java

```diff
@@ -156,9 +156,10 @@ public static boolean checkBackendAvailability() {
             if (rs.getExitCode() != 0) {
                 return false;
             }
-            LOG.info("Doris backend status:\n{}", rs.getStdout());
-            return rs.getStdout()
-                    .contains("*************************** 1. row ***************************");
+            String output = rs.getStdout();
+            LOG.info("Doris backend status:\n{}", output);
+            return output.contains("*************************** 1. row ***************************")
+                    && !output.contains("AvailCapacity: 1.000 B");
```

Comment:
There's a Doris startup stage where `SHOW BACKENDS;` outputs rows while its available disk capacity is zero.

```
*************************** 1. row ***************************
              BackendId: 10004
                   Host: 127.0.0.1
                  Alive: true
                         (...)
              TabletNum: 0
       DataUsedCapacity: 0.000 
     TrashUsedCapcacity: 0.000 
          AvailCapacity: 1.000 B
          TotalCapacity: 0.000
                         (...)
```

Though backend declares itself as "alive", any DDL operations will fail due to a no-disk-available error. Explicitly waiting for disk to initialize fixes this problem.